packages/cypress-commands/src/commands/login.js

```diff
@@ -0,0 +1,57 @@
+const LOGIN_END_POINT = 'dhis-web-commons-security/login.action'
```

Comment:
There was a discussion about this on Slack that I didn't catch the conclusion of, but `dhis-web-commons-security/login.action` gives me a redirect to `dhis-web-commons/security/login.action`. Could we double check that this is indeed the correct URL to use here?

Reply:
This is the correct endpoint-

`dhis-web-commons-security` is a Spring API endpoint which only accepts POST requests

`dhis-web-commons/security` is the STRUTS login app (just the UI, it POSTs to Spring on login)

Reply:
Definitely room for improvement there, which we will address when replacing the web portal

Reply:
Perfect, thanks. Glad to have it documented here in the PR. :+1:

Reply:
the `web-commons-security` is a POST only endpoint to be used for authentication. The `web-commons/security` is for the old Struts framework. So my understanding is that the former is the correct one for our apps to authenticate.

Reply:
https://dhis2.slack.com/archives/C1XNHEY0L/p1603723486038300